src/test/clojure/xtdb/metrics_test.clj

```diff
@@ -56,3 +61,16 @@
       (Thread/sleep 200))
 
     (t/is (= 2.0 (.count ^Counter (.counter (.find registry "tx.error")))))))
+
+(t/deftest test-total-and-active-connections
+  (let [node (xtn/start-node tu/*node-opts*)
+        registry ^CompositeMeterRegistry (tu/component node :xtdb.metrics/registry)
+        _ (.add registry (SimpleMeterRegistry.))
+        conn1 (jdbc/get-connection node)
+        conn2 (jdbc/get-connection node)]
+    (jdbc/execute! conn1 ["SELECT 1"])
+    (.close conn1)
+
+    ;; We have a connection open so this should be equal to 1.0
+    (t/is (= 1.0 (.value ^Gauge (.gauge (.find registry "pgwire.active_connections")))))
+    (t/is (= 2.0 (.count ^Counter (.counter (.find registry "pgwire.total_connections")))))))
```

Comment:
Could you write this with `with-open`, where the second `with-open` contains the assertions.

Reply:
Done